### PR TITLE
[#2379] fix(hive-ci-image): Let user hive be able to access MySQL

### PR DESCRIPTION
**Pull request by charliecheng630**
### What changes were proposed in this pull request?

Let user hive be able to access MySQL in gravitino-ci-hive container.

### Why are the changes needed?

Fix: #2379 

### Does this PR introduce _any_ user-facing change?

No

### How was this patch tested?

`./build-docker.sh --platform linux/arm64 --type hive --image datastrato/gravitino-ci-hive --tag 0.1.9`

### CODE REVIEW

File: docs/docker-image-details.md [hunk 1]
```diff
@@ -82,6 +82,9 @@ You can use this kind of image to test the catalog of Apache Hive.
 
 Changelog
 
+- gravitino-ci-hive:0.1.9
+  - Let user `hive` be able to access MySQL.
```
Comment (mchades) [inline docs/docker-image-details.md:86]: As there was no new image release after the last update, version 0.1.9 simply clears the cache compared to 0.1.8.

So here should be: `remove cache after installing packages`

**Comment by mchades** [comment]
@danhuawang Could you please help to test these changes in your environment, thanks!

**Comment by danhuawang** [comment]
> @danhuawang Could you please help to test these changes in your environment, thanks!

It's ok in my test environment.

**Comment by jerryshao** [comment]
@mchades please help to continue this PR.

**Comment by mchades** [comment]
The version of `gravitino-ci-hive` also need to update to 0.1.9 in `build.gradle.kts`(in the modules: catalog-hadoop, catalog-hive, catalog-lakehouse-iceberg, integration-test)

**Comment by mchades** [comment]
> > @danhuawang Could you please help to test these changes in your environment, thanks!
> 
> It's ok in my test environment.

The modified Dockfile also passed tests in my environment.

Afer @charliecheng630 address the comments, I will push the new image `gravitino-ci-hive:0.1.9` and rerun tests of this PR

**Review by mchades — APPROVED** [review]
Excellent contribution, looking forward to more in the future!

**Pull request merged**